### PR TITLE
Pagination to volunteers page

### PR DESCRIPTION
**Pull request by RaghavPro**
This is for #6 

There's a pagination helper class which takes 3 arguments. Total results count, current page number and  result limit per page.
A macro which takes a pagination class object and renders the UI.

This is how it looks. For this image limit was set to 1. By default it's 50.
screen shot 2015-08-23 at 02 52 32

### CODE REVIEW

**Comment by anandology** [comment]
@devilankur18 suggests that this shouldn't implemented in our code and it looks like [Flask-paginate][1] solves our needs pretty well.

[1]: http://pythonhosted.org/Flask-paginate/

**Comment by RaghavPro** [comment]
Using `flask-paginate` now.

File: cleansweep/volunteers/views.py [hunk 1]
```diff
@@ -14,7 +15,18 @@ def init_app(app):
...
-    return render_template("volunteers.html", place=place)
+    try:
+        page_arg_value = int(request.args.get('page', 1))
+        page = page_arg_value if page_arg_value > 0 else 1  # You never know...
+    except ValueError:
+        page = 1
+    total_count = place.get_member_count()
+    limit = 50
+
+    pagination = Pagination(total=total_count, page=page, per_page=limit,
```
Comment (devilankur18) [inline cleansweep/volunteers/views.py]: @RaghavPro I prefer pagination logic  html file rather than controller. @anandology Any thoughts ?

Comment (anandology) [inline cleansweep/volunteers/views.py]: Using `Pagination` object seems to be a standard practice in Django and Flask. I'm okay with that, but I would prefer the try-except to go to some utils module. Merging it for now.

**Comment by anandology** [comment]
Thanks @RaghavPro. Merged the PR.

**Pull request merged**